sites/cnfpt/src/backend/base/cache.py

```diff
@@ -0,0 +1,84 @@
+"""
+Plugin for django-redis that supports redis sentinel.
+Credits :
+ - https://github.com/lamoda/django-sentinel
+ - https://github.com/KabbageInc/django-redis-sentinel
+"""
+
+from datetime import timedelta
+
+from django.utils.cache import (
+    add_never_cache_headers,
+    patch_response_headers,
+    patch_vary_headers,
+)
+from django.utils.timezone import now
+
+from cms.cache import _get_cache_version, _set_cache_version
+from cms.cache.page import _page_cache_key
+from cms.constants import EXPIRE_NOW, MAX_EXPIRATION_TTL
+from cms.toolbar.utils import get_toolbar_from_request
+from cms.utils.conf import get_cms_setting
+
+
+def set_page_cache(response):
```

Comment:
I'm thinking that this may be hard to maintain and maybe there is a way with a context manager to call the original function patching the user. Even more DYI but this is hopefully temporary waiting for DjangoCMS to accept our change. What do you think?

Reply:
I agree with you on this : this may be hard to maintain and we'll have to follow django CMS's changes carefully.

But I really don't like to patch the user before calling the original function. We'll patch an attribute that has nothing to do with caching and this might lead to dreadful debugging sessions if the behavior of the parent functions changes in the future or if we introduce a bug.

And I don't think it's worth submitting this patch to DjangoCMS. I understand their choice : in most cases, logged user are not served the exact same content. We are a special case. Maybe they could add an option to disable this, but it has to be handled **VERY** carefully to not expose any personal user data to other users because of the page cache.

Reply:
Following your suggestions, I added `raincoat` as a new linter :-)